### ISSUE TITLE
Remodelling of terms associated with invalid unit of Milligrams of carbon per milligram of chlorophyll per hour

### ISSUE THREAD

**Issue by gwemon** [nvs-vocabs/P01#267]
The following units need to be deprecated: https://vocab.nerc.ac.uk/collection/P06/current/UCCL/
This is because "carbon" and "chlorophyll" should not be part of the unit definition.
However before we can deprecate the unit and replace it with Milligrams per milligram per hour, the associated P01 codes need to be remodelled. There are 14 P01 codes associated with this unit:

CMAXPIP2 | Calcification maximum for the   water body [particulate >0.4/0.45um phase] by PvI incubation, filtration   and counting residue and differencing results from acidified and unacidified samples
SECXPIP2 | Calcification maximum standard   error for the water body [particulate >0.4/0.45um phase] by PvI   incubation, filtration and counting residue and differencing results from   acidified and unacidified samples
SFPXPIPE | Photosynthetic maximum for the   water body [particulate 0.2-1um phase] by PvI Incubation, cascade filtration   and counting residue
SFPXPIPD | Photosynthetic maximum for the   water body [particulate 1-5um phase] by PvI Incubation, cascade filtration   and counting residue
PMAXPIP5 | Photosynthetic maximum for the   water body [particulate >0.2um phase] by PvI incubation, filtration and   counting residue
PMAXPIP2 | Photosynthetic maximum for the   water body [particulate >0.4/0.45um phase] by PvI incubation, filtration   and counting residue
PMAXPIP6 | Photosynthetic maximum for the   water body [particulate >20um phase] by PvI incubation, filtration and   counting residue
PMAXPIP7 | Photosynthetic maximum for the   water body [particulate >2um phase] by PvI incubation, filtration and   counting residue
SFPXPIPA | Photosynthetic maximum for the   water body [particulate >5um phase] by PvI incubation, filtration and   counting residue
PMAXPIP1 | Photosynthetic maximum for the   water body [particulate >GF/F phase] by PvI incubation, filtration and   counting residue
PMAXPIPX | Photosynthetic maximum for the   water body [particulate >unknown phase] by PvI incubation, acidification   to remove residual inorganic 14C and counting residue
PMAXPIPZ | Photosynthetic maximum for the   water body [particulate >unknown phase] by PvI incubation, filtration and   counting residue
SEPXPIP2 | Photosynthetic maximum standard   error for the water body [particulate >0.4/0.45um phase] by PvI   incubation, filtration and counting residue
SEPXPIP1 | Photosynthetic maximum standard   error for the water body [particulate >GF/F phase] by PvI incubation,   filtration and counting residue

</body>

</html>

**Comment by ahpanton** [comment]
Is it worth looking at the related terms that use the P06 UCLC 'Milligrams of carbon per (microEinstein per square metre per second) per milligram of chlorophyll per hour' at the same time? Or should I create a new ticket for this?

There are 20 terms with the P06 UCLC

**Comment by gwemon** [comment]
Hi @ahpanton I think it would be better to keep it separate because the use of "microEinstein" brings another issue that makes the solution more complex...

**Comment by ahpanton** [comment]
Hi @gwemon very true. I'll put those on hold for now and focus on these 14 terms

**Comment by ahpanton** [comment]
Remodelling terms using terms recently created for a PvI dataset:

S06 = S0600346 'Light-saturated maximum rate'
S29 = PE001569 'carbon uptake' or PE001570 'calcification'
S02 = S02039 'per hour per unit mass of chlorophyll-a in the'

plus appropriate terms for particle size and methodology

**Comment by ahpanton** [comment]
[Pmax_remodel.xlsx](https://github.com/user-attachments/files/26898848/Pmax_remodel.xlsx)

**Comment by ahpanton** [comment]
Attached above is spreadsheet used for remodelling planning
